hoomd/logging.py

```diff
@@ -102,9 +102,9 @@ def update_cls(self, cls):
 class Logger(SafeNamespaceDict):
     '''Logs HOOMD-blue operation data and custom quantities.'''
```

Comment:
Document parameters to `__init__` and give a more detailed explanation of `Logger`.

Reply:
This is covered in #679.